.github/workflows/migration-tests.yml

```diff
@@ -32,7 +32,7 @@ env:
     TF_ACC: 1
     TF_LOG: ${{ vars.LOG_LEVEL }}
     ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
-    ACCTEST_REGEX_RUN: "^TestAccMigration"  
+    ACCTEST_REGEX_RUN: "^TestMig"  
```

Comment:
I remember a PR where we were migrating from regex to package name 🤔 Why here we still have the regex?

Reply:
Our tests have 2 dimensions: resource/package and type (acc, mig, unit). 

Before: we ran always all packages and used regex for both dimensions, e.g. TestAccSearchIndex meant Acceptance tests for search_index resource.

Now: resource/package is chosen directly and regex is for the test type.

Reply:
Thank you!